lib/Bloc/authentication.dart

```diff
@@ -16,9 +18,11 @@ class Authentication {
     User? user = FirebaseAuth.instance.currentUser;
 
     if (user != null) {
-      print("USER EXISTS");
-      print(user.uid);
-      userID = user.uid;
+
+      Log.info("USER EXISTS");
+      Log.info(user.uid);
+
+      SharedPref.setValue('USER_UID',user.uid);
```

Comment:
aren't there keys for this instead of hardcoding the value here?

Reply:
done